src/main/java/rs/teslaris/core/dto/document/SoftwareDTO.java

```diff
@@ -0,0 +1,19 @@
+package rs.teslaris.core.dto.document;
+
+import javax.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class SoftwareDTO extends DocumentDTO {
+
+    private String internalNumber;
+
+    @Positive(message = "Publisher id cannot be a negative number.")
```

Comment:
is this a message for logs or it can be seen by users?

Reply:
Only visible if you do an API call on you own, never seen by regular user.